lib/fizzy/module.hpp

```diff
@@ -74,5 +74,13 @@ struct Module
     {
         return !memorysec.empty() || !imported_memory_types.empty();
     }
+
+    bool is_global_mutable(GlobalIdx idx) const noexcept
```

Comment:
Hmm, shouldn't we have a unit test for this?

Reply:
For the sake of progress I'm merging this, but perhaps we should add unit tests for each module helper in a new PR>